src/containers/LandingPageContainer.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
```

Comment:
Is this component used anywhere? Seems like it can be deleted

Reply:
Good find!